scala/private/rule_impls.bzl

```diff
@@ -79,6 +80,8 @@ def compile_scala(
     args.add("--Manifest", manifest)
     args.add("--PrintCompileTime", print_compile_time)
     args.add("--ExpectJavaOutput", expect_java_output)
+    if jdepsPath != None:
```

Comment:
When `jdepsPath` can be `None`? Maybe there is no need for `None` and `null` checks?